frontend/packages/knative-plugin/src/components/test-function/TestFunction.tsx

```diff
@@ -46,27 +28,34 @@ const TestFunction: React.FC<TestFunctionProps> = ({ service, cancel, close }) =
       },
     },
     response: {
-      url: '',
       status: '',
-      statusCode: 0,
-      headers: {
-        'content-type': ['application/json'],
-        'ce-type': ['boson.fn'],
-        'ce-source': ['/boson/fn'],
-        'ce-hello': ['World', 'sds'],
-        'ce-lorem': ['Ipsum'],
-      },
-      body: '{"message": "Response Data!"}',
+      statusCode: null,
+      header: {},
+      body: '',
     },
     endpoint: {
       url: '',
+      private: false,
     },
   };
 
   const handleSubmit = (values: FormikValues, action: FormikHelpers<FormikValues>) => {
-    // eslint-disable-next-line no-console
-    console.log('!!!Invoking Function!!!');
-    action.setStatus({ error: '' });
+    action.setSubmitting(true);
```

Comment:
Just remove set `setSubmitting` state here because the view changes directly to the "loading state".

Reply:
Done :)